core/chains/evm/txmgr/eth_resender.go

```diff
@@ -112,11 +117,25 @@ func (er *EthResender) resendUnconfirmed() error {
 		if err != nil {
 			return errors.Wrap(err, "failed to FindEthTxAttemptsRequiringResend")
 		}
+		oldestAttempt, exists := findOldestUnconfirmedAttempt(attempts)
+		if exists {
+			// Wait at least 2 times the EthTxResendAfterThreshold to log critical with an unconfirmedTxAlertDelay
+			if (time.Since(oldestAttempt.CreatedAt) > er.config.EthTxResendAfterThreshold()*2) &&
+				time.Since(er.lastAlertTimestamps[k]) >= unconfirmedTxAlertDelay {
```

Comment:
nit: Move this check to the very beginning.
If last alert time is still within the unconfirmedTxAlertDelay interval, then short circuit and skip everything.